src/manage/commands.py

```diff
@@ -353,8 +353,8 @@ def __init__(self, args, root=None):
                 set_next = a.lstrip("-/").lower()
                 try:
                     key, value, *opts = cmd_args[set_next]
-                except LookupError:
-                    raise ArgumentError(f"Unexpected argument: {a}")
+                except KeyError:
```

Comment:
Is it possible to raise IndexError (or LookupError directly) here?

Reply:
An unpacking error will raise IndexError, which should never happen, so it should bubble out as an internal error rather than unexpected argument.